go.mod

```diff
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/ipfs/go-cid v0.0.7
-	github.com/ipld/go-ipld-prime v0.12.0
+	github.com/ipld/go-ipld-prime v0.0.0-20211007175251-9b20a0b17af8
```

Comment:
Is the plan to wait for the next release of `go-ipld-prime` before merging this?

Reply:
I'll chat with @oed and see how to proceed with releasing this. Worst case, I could clone @warpfork's code with credit and a TODO to use the official go-ipld-prime code once it is released.

Reply:
Just checked with @warpfork. I'll wait till his [PR](https://github.com/ipld/go-ipld-prime/pull/264) is merged and refer directly to it until a release is available for me to refer to.